functions-eventbrite.php

```diff
@@ -1,121 +1,220 @@
 <?php
+
 /**
  * Eventbrite API
  *
  */
 
-function tna_ebapi_url( $organiser, $category, $token ) {
-
-	$url = 'https://www.eventbriteapi.com/v3/events/search/?sort_by=date&organizer.id='.$organiser.$category.'&token='.$token.'&expand=ticket_classes';
+class Simple_Eventbrite_List {
 
-	return $url;
-}
+	/**
+	 * @param $organiser
+	 * @param $category
+	 * @param $token
+	 *
+	 * @return string
+	 */
+	public function url( $organiser, $category, $token ) {
 
-function tna_ebapi_get_eventbrite_json( $url ) {
+		$url = 'https://www.eventbriteapi.com/v3/events/search/?sort_by=date&organizer.id=' . $organiser . $category . '&token=' . $token . '&expand=ticket_classes';
 
-	if ( !class_exists('WP_Http') ) {
-		include_once( ABSPATH . WPINC . '/class-http.php');
+		return $url;
 	}
 
-	$request = new WP_Http;
-	$result = $request->request( $url );
+	/**
+	 * @param $result
+	 *
+	 * @return bool
+	 */
+	public function check_result( $result ) {
+
```

Comment:
Yep, makes sense. Thanks, but I kinda choose to ignore this. My IDE is setup in a way that this doesn't give me issues.